manifests/0000_90_kube-controller-manager-operator_05_cluster-policy-controller-role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: cluster-policy-controller-leader-lock
```

Comment:
if you don't bind this here, move it with the openshift-controller-manager

Reply:
fixed